cloud/interfaces.go

```diff
@@ -70,4 +70,8 @@ type ClusterDescriber interface {
 	NodeSubnet() *infrav1.SubnetSpec
 	ControlPlaneSubnet() *infrav1.SubnetSpec
 	RouteTable() *infrav1.RouteTable
+	ClientID() string
```

Comment:
Should this go in Authorizer above instead?

Reply:
moved 👍